src/main/java/doit/jpastudy2/JpaRepository/Synergy.java

```diff
@@ -0,0 +1,27 @@
+package doit.jpastudy2.JpaRepository;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class Synergy {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "Id_synergy")
+    private Long id;
+
+    private String name;
+
+    private Integer max_activate;
```

Comment:
디테일하게 필드 명 적어주신 부분 좋습니다!
다만 자바에서는 이러한 필드명은 대부분 CamelCase로 작성하는 것을 권장하고 있습니다.
이렇게 CamelCase로 작성한 필드들에 대해서는 jpa를 통해 MySQL로 전달될 때 snake_case로 자동으로 변경됩니다.
ex) JAVA : maxActivate -> JPA -> MySQL : max_activate

따라서 아래처럼 작성하는 것이 보편적이니 참고하시면 될 것 같아요.
```suggestion
    private Integer maxActivate
```